01_todo_app/docker-compose.yml

```diff
@@ -4,6 +4,7 @@ services:
   db:
     build: db
     image: todo-app-db:latest
+    platform: linux/amd64
```

Comment:
Where does this one come from? I don't see this in the official specs. Also, this whole file could be updated to docker-compose v3 (as right now it's 2)

Reply:
It was to make everything run on my M1 macbook which is ARM64. It doesn't change anything except make it possible for ARM devices to emulate a AMD64 container so it can run the service.

`platform` is removed from v3 (can't recall if from a certain point or all of them)

Reply:
Ow, and it also makes sure that the images are built via docker-compose is in the right architecture. If I for example build the images without this parameter, then it builds ARM docker images which are converted to GVMI and when a provider tries to run it, they get an `exec error`